ecommerce-ui/src/ecommerce/Rental.js

```diff
@@ -0,0 +1,99 @@
+import React, { Component } from 'react';
+import './Rental.css';
+// import Cart from './ShoppingCart';
+
+class Rental extends Component {
+    constructor(props) {
+        super(props);
+        this.state = {
```

Comment:
move this state and addItem to the parent component.   Individual rental shouldn't be keeping track of the contents of the shopping cart